services/wallet/service_test.go

```diff
@@ -0,0 +1,188 @@
+package wallet
+
+import (
+	"testing"
+	"time"
+
+	should "github.com/stretchr/testify/assert"
+	must "github.com/stretchr/testify/require"
+	"gopkg.in/square/go-jose.v2"
+	"gopkg.in/square/go-jose.v2/jwt"
+)
+
+func TestParseClaims(t *testing.T) {
```

Comment:
This is a test for future helper that would parse claims. For now, added it to prove that claims can in fact be parsed into a struct.